ledger/ledger-api-test-tool-on-canton/BUILD.bazel

```
@@ -60,6 +60,9 @@ conformance_test(
         "@grpcurl_nix//:bin/grpcurl",
         "@jq_dev_env//:jq",
     ],
+    # FIXME: drop the following line once canton-test-runner supports LF 1.12
+    lf_versions = ["legacy"],
+    # lf_versions = ["stable", "latest"],
```

```suggestion
    lf_versions = ["stable", "latest"],
```
@oliverse-da could you applied the suggestion when updating canton lib ?